nafas/params.py

```diff
@@ -38,6 +38,8 @@
 
 BAD_INPUT_MESSAGE = "[Error] Bad input!"
 
+CONFIG_LOAD_ERROR_MESSAGE = "[Error] Failed to load configuration file!"
```

Comment:
`[Error] Failed to load the configuration file!`

Reply:
fixed in https://github.com/sepandhaghighi/nafas/pull/129/commits/c2903f3538da923951f88c2fbdc2565868d13825